src/Powercord/plugins/pc-commands/commands/disable.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    command: 'disable',
+    description: 'Disable a plugin',
+    usage: '{c} [ Plugin Name ]',
```

Comment:
Although this is very minor, the given argument should be displayed as "plugin ID" (not "plugin name") to avoid confusion.

_This change should be reflected inside of "enable.js" as well!_

```suggestion
    usage: '{c} [pluginID]',
```